code/datums/jobs.dm

```diff
@@ -2426,48 +2467,6 @@ ABSTRACT_TYPE(/datum/job/special/halloween/critter)
 		..()
 		src.access = get_all_accesses() + access_centcom
 
-// Use this one for late respawns to deal with existing antags. they are weaker cause they dont get a laser rifle or frags
-/datum/job/special/nt_security
-	linkcolor = "#3348ff"
-	name = "Nanotrasen Security Consultant"
-	limit = 1 // backup during HELL WEEK. players will probably like it
-	unique = TRUE
-	wages = PAY_TRADESMAN
-	trait_list = list("training_security")
-	requires_whitelist = TRUE
-	requires_supervisor_job = "Head of Security"
```

Comment:
Not against letting NTSC be chosen roundstart, but DNM unless supervisor requirement is re-implemented for roundstart roles and put back on the job datum.